backend/server.py

```
@@ -7964,7 +7964,7 @@ async def scheduler_force_cleanup(user: dict = Depends(get_current_user)):
         return {"ok": True, **summary}
     except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)